src/App.php

```diff
@@ -57,4 +57,17 @@ public function clearRecentDocuments(): void
     {
         $this->client->delete('app/recent-documents');
     }
+
+    public function openAtLogin(?bool $open = null): bool
```

Comment:
Could this be the following as thats how it seems to function anyway?
```suggestion
    public function openAtLogin(bool $open = true): bool
```

I think this makes it clearer that by calling the method without providing a boolean, it defaults to open. :) 

That then removes the requirement for the null-check in the method also.

Reply:
Fair enough! Approved!